### ISSUE TITLE
Shuffle gt_boxes

### ISSUE THREAD

**Issue by LiewFeng** [jwyang/faster-rcnn.pytorch#825]
Hi @severian @jnhwkim @temerick @EMCP @cclauss , I have a question about shuffling gt_boxes. 

In lib/roi_data_layer/roibatchLoader.py, line 72-74:
`
if self.training:
        np.random.shuffle(blobs['gt_boxes'])
        gt_boxes = torch.from_numpy(blobs['gt_boxes'])
`

I wonder why do we need to shuffle the gt_boxes. Thanks a lot!
